trusted.ci.jenkins.io.tf

```diff
@@ -21,7 +26,11 @@ resource "azurerm_resource_group" "trusted_ci_jenkins_io_controller" {
   location = var.location
   tags     = local.default_tags
 }
-
+resource "azurerm_resource_group" "trusted_ci_jenkins_io_permanent_agents" {
```

Comment:
Is a new resource group needed?

Why not putting it in the existing "trusted" resource groupe?

Reply:
as we had a group for the ephemeral, we decided at the beginning to have 3, I would rather have only one, the trusted one ... will talk this with @dduportal  next week and to a subsequent PR if needed.

Reply:
As per https://github.com/jenkins-infra/helpdesk/issues/3486#issuecomment-1540067039, issue updated to explain why.

TL;DR; different lifecycles driven by adding locks (o not).